boards/arm/s32k1xx/rddrone-bms772/Kconfig

```diff
@@ -0,0 +1,184 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+if ARCH_BOARD_RDDRONE_BMS772
+
+config S32K1XX_NRSTCHECK_PROCFS
+	bool "RDDRONE-BMS772 nRST Check PROCFS entry"
+	default n
+	depends on FS_PROCFS
+	select FS_PROCFS_REGISTER
+	---help---
+		Self-test to check that the MCU reset pin is connected to an
+		external pull-up resistor (or an IC that pulls the pin high).
+		The input logic level is registered as a PROCFS entry.
+
+config SMBUS_SBD
+	bool "SMBus Smart Battery Data slave"
+	default n
+	depends on I2C_SLAVE
+	---help---
+		Simple SMBus/I2C slave providing battery management data,
+		specifically implementing a subset of the Smart Battery Data
+		Specification, Revision 1.1
+
+menu "RDDRONE-BMS772 PM configuration"
+depends on PM
+
+menu "PM configuration for PM_STANDBY mode"
+
+choice
+	prompt "MCU power mode"
+	default VLPR_STANDBY
+
+config RUN_STANDBY
+	bool "RUN mode"
+	---help---
+		The MCU should be in RUN mode in the PM_STANDBY mode
```

Comment:
Maybe the description name of this mode should be more clear: i.e.: "Keep in RUN mode during PM Standby"

Reply:
I agree. I changed the description and help text. We plan to revisit all of these PM-related options in the near future anyway, as it is very difficult to understand this configuration for a new user.